xml/System.Threading/Thread.xml

```diff
@@ -94,7 +94,7 @@
   
  The <xref:System.Threading.Thread> constructors can take either of two delegate types, depending on whether you can pass an argument to the method to be executed:  
   
--   If the method has no arguments, you pass a <xref:System.Threading.ThreadStart> delegate to  the constructor. It has the signature:  
+- If the method has no arguments, you pass a <xref:System.Threading.ThreadStart> delegate to the constructor. It has the signature:  
```

Comment:
Not relevant to your change, but this statement doesn't make sense to me. "If the method has no arguments" followed by "you pass a <type> delegate to the constructor" are contradictory; if the method has no arguments, nothing could be passed into it. Perhaps it could just say "If the method has no arguments, it has the following signature:".

Reply:
All of the overloads of the constructors have arguments, I guess what it's referring to is whether there are arguments to the callback. Maybe it can say "if the callback has no arguments" and that would be more clear?

Reply:
Perhaps the previous sentence adds enough context to say what "the method" is referring to. Maybe it's fine as is.